src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    private static final String CONSOLE_CLOSE_COMMAND = "q";
+    private static final String COMMAND_SPLITTING_SYMBOL = " ";
+    BetDao betDao = new BetDaoImpl();
```

Comment:
access modifier

Reply:
What about userDao?

Reply:
But we don't need to use UserDao in our console handler. I can make object of it, but I cant use it here. Or should I create separate handler for User input?
Access modifier added